fluent/handler.py

```diff
@@ -136,13 +136,15 @@ def __init__(self,
                  port=24224,
                  timeout=3.0,
                  verbose=False,
-                 buffer_overflow_handler=None):
+                 buffer_overflow_handler=None,
+                 nanosecond_precision=False):
```

Comment:
I'd rather leave such a rather drastic change to the actual maintainer. Personally, I rather dislike using **kwargs here because it makes it easy to miss (and hide) typos. It might be a matter of style, though.